apps/dashboard/public/drop/README.md

```diff
@@ -0,0 +1 @@
+Medias in this folder are used for /drop/<slug>
```

Comment:
Remove this folder, move the media in app router folder next to the page and use `import ...` syntax for importing the image